plugin/trino-tpcds/src/main/java/io/trino/plugin/tpcds/TpcdsNodePartitioningProvider.java

```diff
@@ -53,10 +55,17 @@ public TpcdsNodePartitioningProvider(NodeManager nodeManager, int splitsPerNode)
     public ConnectorBucketNodeMap getBucketNodeMap(ConnectorTransactionHandle transactionHandle, ConnectorSession session, ConnectorPartitioningHandle partitioningHandle)
     {
         Set<Node> nodes = nodeManager.getRequiredWorkerNodes();
-        checkState(!nodes.isEmpty(), "No TPCDS nodes available");
-
-        // Split the data using split and skew by the number of nodes available.
-        return createBucketNodeMap(toIntExact((long) nodes.size() * splitsPerNode));
+        checkState(!nodes.isEmpty(), "No TPCH nodes available");
+        List<Node> sortedNodes = nodes.stream()
```

Comment:
add a comment why sorting is needed (for sake of future readers). And link to TpcdsSplitManager.
Add cross comment to.

And also in Tpch*